src/features/reviews/utils/queueReviews.js

```diff
@@ -22,11 +21,16 @@ const queueReviews = (reviews, reviewTypesHistory, queue) => {
       newReviewTypeHistory,
     ] = pickReviewAndType(_reviews, _reviewTypesHistory);
 
+    // find the review's subject
+    const subject = subjects.find(subject => subject.id === review.data.subject_id);
```

Comment:
Same comment as above 👀 

```suggestion
    const subject = subjects.find(subject => subject.id === _.get(review, "data.subject_id"));
```